travis-results-1.txt

```diff
@@ -0,0 +1,28 @@
+Cleaning the drupal-build directory
+Building the drupal files
+Processing welcome
+Processing release_notes
+Processing architecture
+Processing installing
+Processing updating
+Processing post_installation_configuration
+Processing support
+Processing web_console
+Processing security
+Processing authentication
+Processing networking
+Processing storage
+Processing registry
+Processing operators
+Processing builds
+Processing pipelines
+Processing openshift_images
+Processing applications
+Processing machine_management
+Processing nodes
+Processing logging
+Processing monitoring
+Processing metering
+Processing scalability_and_performance
+Processing backup_and_restore
+[31m[1mERROR (backup_and_restore/backing-up-etcd.adoc): "xref:../welcome/index-page.adoc#welcome-index[Welcome page]" appears to try to reference a file not included in the "openshift-enterprise" distro[0m
```

Comment:
First error is caught with only
```
python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.5 --no-upstream-fetch > travis-results-1.txt
```

This error is for the incorrect xref to the welcome page.